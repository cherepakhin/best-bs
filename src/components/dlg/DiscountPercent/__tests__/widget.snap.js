import React from 'react';
import DlgDiscountPercent from '../widget.js';
import renderer from 'react-test-renderer';
import { CONST_ACTION } from '../../../../constants.js';

const props = {
  doc: {
    n: 1
  },
  modalType: CONST_ACTION.DLG_DISCOUNT_PERCENT
};

describe('DlgDiscountPercent snapshot', () => {
  it('Проверка title', ()=> {
    const dlg = renderer.create(<DlgDiscountPercent {...props}/>).toJSON();
    expect(dlg).toMatchSnapshot();
  });
});
