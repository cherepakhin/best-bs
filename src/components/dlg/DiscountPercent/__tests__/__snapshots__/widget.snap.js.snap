// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DlgDiscountPercent snapshot Проверка title 1`] = `
<div
  className="static-modal"
>
  <div
    className="modal"
    role="dialog"
    style={
      Object {
        "display": "block",
      }
    }
    tabIndex="-1"
  >
    <div
      className="modal-dialog"
    >
      <div
        className="modal-content"
        role="document"
      >
        <div
          className="modal-header"
        >
          <h4
            className="modal-title"
          >
            Введите прооцент скидки
          </h4>
        </div>
        <div
          className="modal-body"
        >
          <form
            className="form-horizontal"
          >
            <div
              className="form-group"
              style={
                Object {
                  "marginBottom": 0,
                }
              }
            >
              <div
                className="col-sm-12"
              >
                <input
                  className="form-control"
                  id="percent"
                  onKeyDown={[Function]}
                  type="number"
                />
              </div>
            </div>
          </form>
        </div>
        <div
          className="modal-footer"
        >
          <button
            className="btn btn-default"
            disabled={false}
            onClick={[Function]}
            type="button"
          >
            Отмена
          </button>
          <button
            className="btn btn-primary"
            disabled={false}
            onClick={[Function]}
            type="button"
          >
            Сохранить
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
